test/testenv/deployment.go

```diff
@@ -96,6 +101,75 @@ func (d *Deployment) GetInstance(name string, instance runtime.Object) error {
 	return nil
 }
 
+// CopyFileToPod copies a file locally from srcPath to the destPath on the pod specified in podName
```

Comment:
Move this to ingest util. Deployment.go has methods strictly related to splunk deployments.

Reply:
This function is generic to copy a file to a pod.  It has nothing ingest specific.  I can move it there though it might require some refactoring.

Reply:
Since the only used for this is during ingestion (at least for now), moved to ingest_utils.go.